Modules/vc-module-export/VirtoCommerce.ExportModule.Data/Security/ExportAuthorizationFilter.cs

```diff
@@ -0,0 +1,67 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace VirtoCommerce.ExportModule.Data.Security
+{
+    /// <summary>
+    /// Implements authorization for several policies, given to action by <see cref="AuthorizeAnyAttribute"/>.
+    /// </summary>
+    public class ExportAuthorizationFilter : IAuthorizationFilter, IAsyncAuthorizationFilter
```

Comment:
Maybe we could call it AnyPolicyAuthorizationFilter, as there is no export specific here? And comment should tell that handler authorizes when any of specified policies is authorized.